examples/pytorch/gin/README.md

```diff
@@ -28,15 +28,22 @@ An experiment on the GIN in customized settings can be run with
 python main.py [--device 0 | --disable-cuda] --dataset COLLAB \
                --graph_pooling_type max --neighbor_pooling_type sum
 ```
+hint: **following the settings like the paper** is essential to reproduce the result reported.
+
+add `--degree_as_nlabel` to  use one-hot encodings of node degrees as node feature vectors
 
 Results
 -------
 
 Run with following with the double SUM pooling way:
```

Comment:
"Run with following with the double SUM pooling way:" This does not hold for all datasets. I think it's better to simply list the commands for all datasets, one command per dataset.

Reply:
I will update this section after testing every dataset.